crates/fluvio-test/src/tests/longevity/producer.rs

```diff
@@ -62,6 +75,13 @@ pub async fn producer(test_driver: TestDriver, option: LongevityTestCase, produc
                 .expect("Producer Send failed");
         }
 
+        if last_flush_time.elapsed() > flush_period {
+            for p in &producers {
+                let _ = p.flush().await;
```

Comment:
Why is this is necessarily? Shouldn't linger perform flush?

Reply:
To hold the current thread until all records are sent reducing the overall pressure.

Reply:
I see.  Since there are too many producers, internal accumulation might be too large until flushing.